ssg/yaml.py

```diff
@@ -48,9 +49,20 @@ def _open_yaml(stream, original_file=None, substitutions_dict={}):
 
         return yaml_contents
     except Exception as e:
+        count = 0
         _file = original_file
         if not _file:
             _file = stream
+        with open(_file, "r") as e_file:
+            lines = e_file.readlines()
+            for line in lines:
+                count = count + 1
+                if re.match(r"^\t+\s+", line) or re.match(r"^\s+\t+", line):
```

Comment:
If there is just a single tab, this will not match. I have found that the YAML spec doesn't allow using tabs for indentation at all.  See https://yaml.org/spec/1.1/#indentation%20space/

Reply:
@jan-cerny thanks. Fixed like how we talked.